docs/howto/manage-web-app-charms/configure-web-app-charm.rst

```diff
@@ -85,8 +85,132 @@ The configuration can be set on the deployed charm using:
 
     juju config <app name> token=<token>
 
-Manage secrets for a 12-factor app charm
-----------------------------------------
+Add a custom action
+-------------------
+
+For certain commands or processes that you will run many times, you can set up
+custom Juju actions to expedite your workflow. Actions are handled in the Ops
```

Comment:
> you will run many times

The "you will" throws me a bit here because of the tone of a how-to. Can we be more descriptive with:

```suggestion
For commands and processes that that see shared or frequent use,
custom Juju actions can expedite your workflow. Actions are handled in the Ops
```